lib/verifier.js

```diff
@@ -101,7 +101,7 @@ Verifier.checkTxProposalBody = function(credentials, txp) {
     var proposalHeader = {
       outputs: outputs,
       message: txp.encryptedMessage || txp.message || null,
-      payProUrl: txp.payProUrl || undefined
+      payProUrl: txp.payProUrl
```

Comment:
yes, both verifier and api should have the same logic. I still think that undefined value for payProUrl should be nullified (in both places).

Btw perhaps you might be interested to follow on this: https://github.com/bitpay/bitcore-wallet-utils/pull/29